src/main/kotlin/net/maizegenetics/phgv2/utils/SeqUtils.kt

```diff
@@ -385,3 +387,88 @@ fun queryAgc(commands:Array<String>):Map<Pair<String,String>,NucSeq> {
     }
     return genomeChromNucSeq
 }
+
```

Comment:
Add function description comment before the function.

Reply:
Ok, I will do that